yt_dlp/extractor/sejmpl.py

```diff
@@ -0,0 +1,219 @@
+            'title': 'Konferencja prasowa - Stanowisko SLD ws. składu nowego rządu',
+            'description': 'SLD - Biuro Prasowe Klubu',
+            'duration': 514,
+            'location': 'sala 101/bud. C',
+            'live_status': 'was_live',
+        },
+        'playlist': [{
+            'info_dict': {
+                'id': 'A6E6D475ECCC6FE5C1257EF90034817F',
+                'ext': 'mp4',
+                'title': 'Konferencja prasowa - Stanowisko SLD ws. składu nowego rządu',
+                'duration': 514,
+            },
+        }],
+    }, {
+        'url': 'https://sejm-embed.redcdn.pl/Sejm10.nsf/VideoFrame.xsp/FED58EABB97FBD53C1258A7400386492',
+        'only_matching': True,
+    }]
+
+    def _real_extract(self, url):
+        if not _HAS_ZONEINFO:
+            raise ExtractorError('zoneinfo is required. Use Python 3.9+', expected=True)
+        term, video_id = self._match_valid_url(url).group('term', 'id')
+        frame = self._download_webpage(
+            f'https://sejm-embed.redcdn.pl/Sejm{term}.nsf/VideoFrame.xsp/{video_id}',
+            video_id)
+        # despite it says "transmisje_arch", it works for live streams too!
+        data = self._download_json(
+            f'https://www.sejm.gov.pl/Sejm{term}.nsf/transmisje_arch.xsp/json/{video_id}',
+            video_id)
+        params = data['params']
+
+        title = data['title'].strip()
+
+        def rfc3339_to_atende(date):
+            date = datetime.datetime.fromisoformat(date)
+            # marks naive as CET/CEST
+            date.replace(tzinfo=ZoneInfo('Europe/Warsaw'))
+            # UTC timestamp with an offset
+            return int((date.astimezone(datetime.timezone.utc).timestamp() - 978307200) * 1000)
+
+        if data.get('status') == 'VIDEO_ENDED':
+            live_status = 'was_live'
+        elif data.get('status') == 'VIDEO_PLAYING':
+            live_status = 'is_live'
+        else:
+            live_status = None
+            self.report_warning(f'unknown status: {data.get("status")}')
+
+        start_time = rfc3339_to_atende(params['start'])
+        # current streams have a stop time of *expected* end of session, but actual times
+        # can change during the transmission. setting a stop_time would artificially
+        # end the stream at that time, while the session actually keeps going.
+        if live_status == 'was_live':
+            stop_time = rfc3339_to_atende(params['stop'])
+        else:
+            stop_time = None
+
+        duration = (stop_time - start_time) // 1000 if stop_time else None
+
+        entries = []
+
+        def add_entry(file, legacy_file=False):
+            if not file:
+                return
+            file = f'https:{file}' if file.startswith('//') else file
+            if not legacy_file:
+                update_url_query(file, {'startTime': start_time})
+                if stop_time is not None:
+                    update_url_query(file, {'stopTime': stop_time})
```

Comment:
You must have missed this when applying the suggestion
```suggestion
                file = update_url_query(file, {'startTime': start_time})
                if stop_time is not None:
                    file = update_url_query(file, {'stopTime': stop_time})
```